lint-staged.config.js

```diff
@@ -0,0 +1,7 @@
+const lint = 'eslint --quiet --fix --cache';
+
+// https://github.com/okonet/lint-staged/issues/825#issuecomment-620018284
+module.exports = {
+  '*.{js}': [lint],
+  '*.{ts,tsx}': [lint, () => 'tsc --skipLibCheck --noEmit'],
+};
```

Comment:
I believe we're able to put this on `package.JSON` right? Just thinking if configs are small perhaps they're fine there? We have others there at least.

Reply:
haha I left exactly the same comment, but it seems there's an issue with that, hence why the comment above (I missed it too)